README.md

```diff
@@ -47,9 +47,9 @@ In order to run this task, couple elements are required within the infrastructur
 |`aws_access_key`|Amazon AWS access key for Terraform. See value format [here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)|`-`|`((aws.access_key))`|`True`|
 |`aws_default_region`|Amazon AWS region to use for Terraform.|`-`|`eu-west-1`|`True`|
 |`aws_secret_key`|Amazon AWS secret key for Terraform. See value format [here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)|`-`|`((aws.secret_key))`|`True`|
-|`config_git_branch`|Branch of the config Git repository.|`-`|`master`|`True`|
+|`config_git_branch`|Branch of the config Git repository.|`-`|`config`|`True`|
```

Comment:
Why changing that?
TBH `master` makes more sense most of the time as you usually create a dedicated git repository to hold the config, the default branch will most of the time be `master`.